MaterialSkin/HTML/material/html/js/queue-page.js

```diff
@@ -94,7 +94,7 @@ function parseResp(data, showTrackNum, index, showRatings, queueAlbumStyle, queu
             for (var idx=0, loop=data.result.playlist_loop, loopLen=loop.length; idx<loopLen; ++idx) {
                 let i = loop[idx];
                 splitMultiples(i);
-                let title = i.title;
+                let title = i.title.toLowerCase().replace(/\W/g, '').includes(i.work.toLowerCase().replace(/\W/g, '')) ? i.title : i.work+SEPARATOR+i.title;
```

Comment:
This should really be:
```javascript
let title = undefined==i.work || i.title.toLowerCase().replace(/\W/g, '').includes(i.work.toLowerCase().replace(/\W/g, '')) ? i.title : i.work+SEPARATOR+i.title;
```

...as only want to do this for items that have work - no point in performing the lowercase conversion, etc., if there is no work.

Also, would be better to add this to a function - perhaps in `utils.js` e.g.

```javascript
function trackTitle(i) {
    return undefined==i.work || i.title.toLowerCase().replace(/\W/g, '').includes(i.work.toLowerCase().replace(/\W/g, '')) ? i.title : i.work+SEPARATOR+i.title;
}
```

Reply:
Yes, after creating this PR I found  I had to put in the defined check, otherwise the JS can error with work not defined.

The function idea is a good one, and I'll try to find where to insert the call for now playing, etc, but it might take me some time. I'd be more than happy (and grateful) if you could take this on! :)